device/dhcp/servers/v4/dhcpv4serverLeases.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    DhcpServerV4.Leases:
+      description: >-
+        Configuration for DHCPv4 Leases.
+      type: object
+      x-status:
+        status: under_review
+      properties:
+        lease_time: 
+          description: >-
+            The Life Time length in seconds that is assigned to a lease if the requesting DHCP client does not specify a specific expiration time.
+          type: integer
+          format: uint32
+          minimum: 300
+          maximum: 30000000
+          default: 86400
+          x-field-uid: 1
+        start_pool_address:
+          description: >-
+            The IP address of the first lease pool.
+          type: string
+          format: ipv4
+          default: 100.1.1.1
+          x-field-uid: 2
+        prefix:
```

Comment:
start_pool_address, prefix, pool_size, step - all fields are related to the **pool**. So, can we add a uniform prefix **pool** for all these fields?